yt_dlp/extractor/fujitv.py

```diff
@@ -13,61 +17,245 @@ class FujiTVFODPlus7IE(InfoExtractor):
-        'skip': 'Video available only in one week',
-    }]
+        {
+            'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810083',
+            'info_dict': {
+                'id': '5d40810083',
+                'ext': 'ts',
+                'title': '#1324 『まる子とオニの子』の巻／『結成！2月をムダにしない会』の巻',
+                'description': 'md5:3972d900b896adc8ab1849e310507efa',
+                'series': 'ちびまる子ちゃん',
+                'series_id': '5d40',
+                'thumbnail': 'https://i.fod.fujitv.co.jp/img/program/5d40/episode/5d40810083_a.jpg',
+            },
+            'skip': 'Video available only in one week',
+        },
+    ]
 
     def _real_extract(self, url):
         series_id, video_id = self._match_valid_url(url).groups()
-        self._request_webpage(HEADRequest(url), video_id)
-        json_info = {}
-        token = self._get_cookies(url).get('CT')
-        if token:
-            json_info = self._download_json(
-                f'https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id={video_id}&is_premium=false',
-                video_id, headers={'x-authorization': f'Bearer {token.value}'}, fatal=False)
-        else:
-            self.report_warning(f'The token cookie is needed to extract video metadata. {self._login_hint("cookies")}')
-        formats, subtitles = [], {}
-        src_json = self._download_json(f'{self._BASE_URL}abrjson_v2/tv_android/{video_id}', video_id)
-        for src in src_json['video_selector']:
-            if not src.get('url'):
-                continue
-            fmt, subs = self._extract_m3u8_formats_and_subtitles(src['url'], video_id, 'ts')
-            for f in fmt:
-                f.update(dict(zip(('height', 'width'),
-                                  self._BITRATE_MAP.get(f.get('tbr'), ()), strict=False)))
-            formats.extend(fmt)
-            subtitles = self._merge_subtitles(subtitles, subs)
+        self._request_webpage(HEADRequest(self._FRONTEND_URL), video_id)
+        token = None
+        cookies = self._get_cookies(self._FRONTEND_URL)
+        if cookies and 'CT' in cookies:
+            token = cookies['CT']['value']
+        if not token:
+            # Fallback: try to fetch from HTML if needed (rare)
+            front_page = self._download_webpage(
+                self._FRONTEND_URL, video_id, note='Downloading front page for CT token', fatal=False,
+            )
+            token = self._search_regex(r'CT=([^;]+);', front_page or '', 'ct token', default=None)
+        if not token:
+            raise ExtractorError('CT token not found')
+
+        json_response = self._download_json(
+            f'{self._API_URL}/apps/api/c/v2/lineup/detail/',
+            series_id,
+            headers={'x-authorization': f'Bearer {token}'},
+            query={'is_premium': 'false', 'lu_id': series_id},
+        )
+
+        episode = traverse_obj(json_response, ('episodes', lambda _, v: str(v.get('ep_id')) == video_id))
```

Comment:
Oops, I forgot the `any` to not get a list back
```suggestion
        episode = traverse_obj(json_response, ('episodes', lambda _, v: str(v.get('ep_id')) == video_id, any))
```